### PR TITLE
WIP: [CFM-873] Log group admin demote action - to be closed, as CFM-874 creates log message

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 , #1061 

https://issuetracker.amplexor.com/jira/browse/CFM-873

### CODE REVIEW

**Comment by ordavidil** [comment]
@anvmn 
I don't see where you actual create the message ...

**Comment by anvmn** [comment]
@ordavidil 
Made few modifications. Now you should see everything.

selection_002

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -149,6 +149,28 @@ function c4m_user_notifications_notify_admins_admin_granted($gid, $uid) {
...
+ *   User id.
+ */
+function _c4m_user_notifications_log_admin_revoked($gid, $uid) {
+  global $user;
+  $message = message_create('c4m_log_group_admin_demoted', array('uid' => $uid));
+  c4m_message_load_tokens__c4m_log_group_admin_demoted($message);
+
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $wrapper->field_group_node->set($gid);
+  $wrapper->field_new_member->set($user->uid);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @anvmn 
not sure where are we using this field.

**Comment by anvmn** [comment]
This PR is dependent on https://github.com/capacity4dev/capacity4more/pull/1061, where message field_related_user is introduced.

**Comment by anvmn** [comment]
Close in favor of #1076, which logs the message, before mailing it.

**Pull request closed without merging**